samples/custom-network-state-stream/build.gradle

```diff
@@ -0,0 +1,51 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+androidExtensions {
+    experimental = true
+    features = ['parcelize']
+}
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.instacart.formula.stopwatch"
```

Comment:
Should this change? Probably doesn't matter, but in case someone wanted to install all the samples at once.

Reply:
Oops, missed during copy-paste.